articles/01-DownloadingTheXRTK.md

```diff
@@ -109,10 +117,10 @@ Advanced developers and those wishing to contribute to the Mixed Reality Toolkit
   ![Symbolic Link Context Menu](../images/DownloadingTheXRTK/SymbolicLinks.png)
 4. Using the symbolic link wizard, choose the source folder for the package you wish you replace with the editable version, and the target path for your project, then import the package.
   ![Symbolic Link Wizard](../images/DownloadingTheXRTK/SymbolicLinkWizard.png)
-    > <font color="orange">**Important:**</font> It's strongly advised to use the `Packages` folder as your target path for all symbolically linked packages.
-5. Repeat step 4 for each additional package you'd like to directly modify.
+    > <font color="yellow">**Important:**</font> It's strongly advised to use the `Packages` folder as your target path for all symbolically linked packages.
```

Comment:
I had made this orange bc the yellow was hard to read on a white background

Reply:
Odd, I didn't actually change that, fixed